packages/core/src/tracing/spanBuffer.js

```diff
@@ -98,14 +89,15 @@ exports.init = function init(config, _downstreamConnection) {
   forceTransmissionStartingAt = config.tracing.forceTransmissionStartingAt;
   transmissionDelay = config.tracing.transmissionDelay;
   batchingEnabled = config.tracing.spanBatchingEnabled;
-  initialDelayBeforeSendingSpans = Math.max(transmissionDelay, minDelayBeforeSendingSpans);
+  minDelayBeforeSendingSpans = Math.max(transmissionDelay, minDelayBeforeSendingSpans);
```

Comment:
Only one config: minDelayBeforeSendingSpans
No need to have two: initial + default